cli/Cargo.toml

```diff
@@ -8,6 +8,11 @@ edition = "2018"
 
 [dependencies]
 clap = "2.33.0"
+tar = "0.4"
+walkdir = "2"
+ignore = "0.4.7"
```

Comment:
super cool crate that supports the .gitignore man page spec and also adds supports for a .ignore file